### DISCUSSION TITLE
Strange Visual glitches

### DISCUSSION THREAD

**Discussion by akashdiphazra** [ghostty-org/ghostty#4062] [Q&A] [answered]
https://github.com/user-attachments/assets/d3ae7b60-e367-47c7-b5c6-c83022469a10

OS : Fedora 41 Gnome 
Ghostty Version : 1.0.1

**Comment by pluiedev** [comment, marked as answer]
Most likely an upstream GTK issue. We don't control the rendering over these UI elements.
